AdminUi/src/AdminUi/ClientApp/package.json

```diff
@@ -4,7 +4,7 @@
     "license": "MIT",
     "scripts": {
         "ng": "ng",
-        "start": "ng serve --port 44480",
+        "start": "ng serve --host 0.0.0.0 --disable-host-check --poll 1000",
```

Comment:
1000 means every second? That sounds like a really short time. Did you experience any implications on performance?

Reply:
I did not. It is not clear how much this should impact performance, but `1000` the most common value out there in the wild. I suggest we keep an eye on this and increase it if need be.